python/examples/raw_message_decode.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+
+from argparse import ArgumentParser
+import os
+import sys
+
+root_dir = os.path.normpath(os.path.join(os.path.dirname(__file__), '..'))
+sys.path.append(root_dir)
+
+from fusion_engine_client.messages.core import *
+from fusion_engine_client.messages import ros
+
+from examples.message_decode import print_message
+
+
+def decode_message(header, data, offset):
+    # Validate the message length and CRC.
+    if len(data) != header.calcsize() + header.payload_size_bytes:
+        return False
+    else:
+        header.validate_crc(data)
```

Comment:
does this actually do anything if the crc is bad?

Reply:
Yes, this function raises an exception on failure.